index.src.html

```diff
@@ -309,21 +319,27 @@ <h3 dfn export id="referrer-policy-same-origin">"<code>same-origin</code>"</h3>
     <a><code>Referer</code></a> header.
   </div>
 
+  <div class="example">
+    If a document at <code>https://example.com/page.html</code> sets a policy of
+    <a>"<code>same-origin</code>"</a>, and fetches a module script at
+    <code>https://script.example.com</code>, which then fetches a descendant script
+    at <code>https://example.com/descendant.js</code>, the request for the descendant
+    script would send no <a><code>Referer</code></a> header.
```

Comment:
Could you perhaps expand this example to explain that the header will not be sent because the descendant script's current URL is https://example.com/descendant.js and its resolved referrer is https://script.example.com, hence the request is cross-origin-referrer?

Reply:
Done.